test/v8/v8-module/main.cpp

```diff
@@ -238,6 +238,17 @@ static bool perform_string_test(const FunctionCallbackInfo<Value> &info,
     return false;
   }
 
+  if (v8_string->Utf8Length(isolate) != encoded_utf_8_length) {
+    fail(info, "String::Utf8Length returned wrong length: expected %d got %d",
+         encoded_utf_8_length, v8_string->Utf8Length(isolate));
+    return false;
+  }
+
+  if (v8_string->IsExternal()) {
+    fail(info, "String::IsExternal returned true");
+    return false;
+  }
+
```

Comment:
It'd be appropriate to check `IsOneByte`, `ContainsOnlyOneByte`, `IsExternalTwoByte`, and `IsExternalOneByte` here too. That will also help ensure the mangled symbol names are correct. But it'll be easier to do that with the test refactors I've done on #13426, so I think I'll just do it there once this lands in `main`.